cvat-ui/src/components/tasks-page/task-item.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+
+import Text from 'antd/lib/typography/Text';
+import {
+    Col,
+    Row,
+    Button,
+    Icon,
+    Progress,
+    Menu,
+    Dropdown,
+} from 'antd';
+
+import moment from 'moment';
+
+export interface TaskItemProps {
+    task: any;
+}
+
+export interface TaskItemState {
+    // Should we put it into redux state?
+    preview: string;
```

Comment:
I think yes, better to set this value in redux state.

Reply:
Moved to redux state